pkg/testcase/daemonset.go

```diff
@@ -19,12 +19,10 @@ func TestDaemonset(applyWorkload, deleteWorkload bool) {
 	}
 	pods, _ := shared.GetPods(false)
 
-	cmd := fmt.Sprintf(`
-		kubectl get pods -n test-daemonset -o wide --kubeconfig="%s" \
-		| grep -A10 NODE | awk 'NR>1 {print $7}'
-		`,
-		shared.KubeConfigFile,
-	)
+	cmd := "kubectl get pods -n test-daemonset" +
+		` -o jsonpath='{range .items[*]}{.spec.nodeName}{"\n"}{end}'` +
```

Comment:
What is the new command doing in contrast to the previous one?

Reply:
So previously it was taking the node name using grep and parsing the output, this change uses pod objects to get the same data it expects…